flows/codelingo/review/review/cli.go

```diff
@@ -166,16 +169,67 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 		return nil, nil, nil, nil, errors.Trace(err)
 
 	}
+	// Build `path` for go projects as "reponame" for a local repo or "github.com/username/reponame" for a github repo
+	// path variable is initially empty and will remain empty for none go repos
+	path := ""
+
+	// Find the current working directory, i.e. where user executed `lingo run review` command on the repo
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, nil, nil, nil, errors.Trace(err)
+
+	}
+	// Check if the repo is a go project
+	goProject, err := isGoProject(wd)
+	if err != nil {
+		return nil, nil, nil, nil, errors.Trace(err)
+	}
+
+	// Check if GOPATH is set
+	goPath := os.Getenv("GOPATH")
+
+	if goPath != "" && goProject {
+		if strings.Contains(wd, "github.com") { // Check if the working directory includes "github.com/username/reponame"
```

Comment:
Can you explain the purpose of lines 192-204? I figured you could do something as simple as:

```go
path = strings.TrimPrefix(wd, goPath)
```